packages/ndla-ui/src/locale/messages-nn.ts

```diff
@@ -258,6 +258,7 @@ const messages = {
     alphabeticSort: 'Fag gruppert alfabetisk',
     subjectGroup: 'Gruppe "{{ category }}"',
     allSubjects: 'Alle fag',
+    myFavoriteSubjects: 'Mine favorittfag',
```

Comment:
Bedre ordlyd på nynorsk
```suggestion
    myFavoriteSubjects: 'Favorittfaga mine',
```